tests/garage/torch/modules/test_gaussian_mlp_module.py

```diff
@@ -0,0 +1,362 @@
+import pickle
+
+import pytest
+import torch
+from torch import nn
+
+from garage.torch.modules.gaussian_mlp_module import GaussianMLPModule
+
+plain_settings = [
+    (5, 1, (1,)),
```

Comment:
please try a larger range of input_dims. problems often occur around the values 1, 2, and more than 2.

Reply:
i think the ranges used in https://github.com/rlworkgroup/garage/blob/master/tests/garage/tf/models/test_gaussian_mlp_model.py should be sufficient.

Reply:
fixed

Reply:
this is a lot of ranges (a lot more than TF) -- perhaps pick some covering subset? we are pushing how long the CI can run right now.

Reply:
on my local machine, it was just taking a few seconds.
so I think it would be fine